<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-2BENLPR" failures="1" tests="1" name="Test" time="86.961" errors="0" timestamp="2024-05-08T17:14:08 IST">
  <testcase classname="testCases.TC_002_LoginTest" name="verify_login" time="14.422">
    <failure type="java.lang.AssertionError" message="Login passed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login passed expected [true] but found [false]
at testCases.TC_002_LoginTest.verify_login(TC_002_LoginTest.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
  <testcase classname="testCases.TC_001_AccountRegistrationTest" name="verify_account_registration" time="0.0">
    <ignored/>
  </testcase> <!-- verify_account_registration -->
</testsuite> <!-- Test -->
